lib/jormungandr/exe/cardano-wallet-jormungandr.hs

```diff
@@ -273,7 +276,25 @@ genesisHashOption = optionT $ mempty
     <> help "Blake2b_256 hash of the genesis block, in base 16."
 
 -- | -- [ARGUMENTS...]
-extraArguments :: Parser [Text]
-extraArguments = some $ argument str $ mempty
+extraArguments :: Parser [String]
+extraArguments = many $ argument jmArg $ mempty
     <> metavar "[-- ARGUMENTS...]"
     <> help "Extra arguments to be passed to jormungandr."
+  where
+    jmArg = do
+        arg <- readerAsk
+        case validate arg of
+            Just err -> readerError err
+            Nothing -> pure arg
+    validate arg
+        | "--genesis-block" `isPrefixOf` arg = Just $
+            "The " <> arg <> " option must be placed before the --"
+        | "--rest-listen" `isPrefixOf` arg = Just $
+            suggestion "--rest-listen"
+        | "--storage" `isPrefixOf` arg = Just $
+            suggestion "--storage"
+        | otherwise = Nothing
```

Comment:
are we missing `--log-level` here? it is also provided automatically on `launch`...

btw. why do we want to have `--log-level info` hardcoded in `launch`?

Reply:
The log level setting was in `config.yaml` before, so I kept it. But I think it's better to decouple the log levels of cardano-wallet and jormungandr, so I have now removed this field.